web_widget_uom_view_precision/readme/USAGE.rst

```diff
@@ -0,0 +1,22 @@
+Set your preferred precision on UOM records.
+
+In the form view declaration, put widget='uom' attribute in the field tag::
+
+    ...
+    <field name="arch" type="xml">
+        <form string="View name">
+            ...
+            <field name="quantity_field" widget="uom"/>
+            ...
+        </form>
+    </field>
+    ...
+
+
+You can also specify the uom field with::
```

Comment:
could you specify what is the default, if no value is specified in ``uom_field`` ?

Reply:
the default is "uom_id", i clarified the description.